packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -161,8 +161,11 @@ export interface DroppableCollectionProps {
   onDropEnter?: (e: DroppableCollectionEnterEvent) => void,
   /** Handler that is called when a valid drag is moved within the drop target. */
   onDropMove?: (e: DroppableCollectionMoveEvent) => void,
-  /** Handler that is called after a valid drag is held over the drop target for a period of time. */
-  onDropActivate?: (e: DroppableCollectionActivateEvent) => void,
+  /**
+   * Handler that is called after a valid drag is held over the drop target for a period of time.
+   * @private
+   **/
+  // onDropActivate?: (e: DroppableCollectionActivateEvent) => void,
```

Comment:
Think you didn't mean to comment this out since you marked it private instead?
```suggestion
  onDropActivate?: (e: DroppableCollectionActivateEvent) => void,
```

Reply:
which do we prefer?

Reply:
k, private gets rid of the TS errors. do we need to stop using this prop internally though? or is it ok to stay?

Reply:
IMO we can keep it for now, so long as the other dnd hooks have it marked as private as well (think useDrop needs to hide it as well).